packages/preact/README.md

```diff
@@ -0,0 +1,114 @@
+# `@appsignal/preact`
+
+- [AppSignal.com website][appsignal]
+- [Documentation][docs]
+- [Support][contact]
+
+The `@appsignal/javascript` integration for Preact v10.0.0-rc.0+ (Preact X). As Preact X is currently pre-release, you should also consider this pre-release functionality.
+
+## Installation
+
+Add the  `@appsignal/preact` and `@appsignal/javascript` packages to your `package.json`. Then, run `yarn install`/`npm install`.
+
+You can also add these packages to your `package.json` on the command line:
+
+```
+yarn add @appsignal/javascript @appsignal/preact
+npm install --save @appsignal/javascript @appsignal/preact
+```
+
+## Usage
+
+### Error Boundary
+
+If you are using React v16 or higher, you can use the `ErrorBoundary` component to catch errors from anywhere in the child component tree.
+
+```jsx
+import React from "react"
+import ReactDOM from "react-dom"
+import Appsignal from "@appsignal/javascript"
+import { ErrorBoundary } from "@appsignal/preact"
+
+const appsignal = new Appsignal({ 
+  key: "YOUR FRONTEND API KEY"
+})
+
+const FallbackComponent = () => (
+  <div>Uh oh! There was an error :(</div>
+)
+
+const App = () => (
+  <ErrorBoundary 
+    instance={appsignal} 
+    tags={{ tag: "value" }} {/* Optional */}
+    fallback={(error) => <FallbackComponent />} {/* Optional */}
+  >
+    { /** Child components here **/}
+  </ErrorBoundary>
+)
+
+ReactDOM.render(<App />, document.getElementById("root"))
```

Comment:
I see Preact is recommending the `render` function on it's own? https://preactjs.com/guide/switching-to-preact#4-simplify-root-render
should we also do that in our docs?

Reply:
Fixed in 17c5cbf.